src/Helper.ts

```diff
@@ -298,76 +265,78 @@ class Helper {
    *
    * The result excludes hidden and disabled entities.
    *
-   * @param {areaEntity} area Area entity.
+   * @param {generic.areaEntity} area Area entity.
    * @param {string} domain The domain of the entity-id.
    *
-   * @return {hassEntity[]} Array of device entities.
+   * @return {generic.hassEntity[]} Array of device entities.
    * @static
    */
-  static getDeviceEntities(area, domain) {
+  static getDeviceEntities(area: generic.areaEntity, domain: string): generic.hassEntity[] {
     if (!this.isInitialized()) {
       console.warn("Helper class should be initialized before calling this method!");
     }
 
     // Get the ID of the devices which are linked to the given area.
-    const areaDeviceIds = this.#devices.filter(device => {
+    const areaDeviceIds = this.#devices.filter((device: any) => {
       return device.area_id === area.area_id;
-    }).map(device => {
+    }).map((device: any) => {
 
       return device.id;
     });
 
     // Return the entities of which all conditions of the callback function are met. @see areaFilterCallback.
     return this.#entities.filter(
-        this.#areaFilterCallback, {
-          area: area,
-          domain: domain,
-          areaDeviceIds: areaDeviceIds,
-        })
-        .sort((a, b) => {
-          /** @type hassEntity */
-          return a.original_name?.localeCompare(b.original_name);
-        });
+      this.#areaFilterCallback, {
+        area: area,
+        domain: domain,
+        areaDeviceIds: areaDeviceIds,
+      })
+      .sort((a: any, b: any) => {
+        /** @type hassEntity */
```

Comment:
```suggestion
      .sort((a: generic.hassEntity, b: generic.hassEntity) => {
```

Reply:
Good catch, but you're way ahead of me. This type was is set to `any` to be able to compile and test the code. All `any` types must be evaluated.
In this case, I've changed the type already locally.

Reply:
Already changed locally.

Reply:
The type was set to any to be able to compile the script to test it for functionality.
Setting the correct type will currently result in a compile error.

`b.original_name` might be undefined while `localeCompare()` expects a string.